### DISCUSSION TITLE
Is there a way to increase system resource useage?

### DISCUSSION THREAD

**Discussion by dreemur99** [LostRuins/koboldcpp#35] [Q&A] [answered]
At the moment it seems to only be using a fraction of the available CPU/RAM my computer has available (I have a high end CPU and a LOT of RAM). I would assume if I could increase the RAM usage 2 or 3 times than generation would scale accordingly. Is there a way to change how much resources KoboldCPP utilizes during generation? I set KoboldCPP's priority in task manager to "realtime" and my computer to "performance mode" and that seemed to give generation speed a small boost but not by much.

**Comment by LostRuins** [comment, marked as answer]
You can set it to use more threads with the --threads parameter.
